app/src/main/java/eu/kanade/tachiyomi/data/database/tables/MangaTable.kt

```diff
@@ -64,6 +66,7 @@ object MangaTable {
             $COL_HIDE_TITLE INTEGER NOT NULL,
             $COL_CHAPTER_FLAGS INTEGER NOT NULL,
             $COL_DATE_ADDED LONG
```

Comment:
Comma is missing
```suggestion
            $COL_DATE_ADDED LONG,
```

Reply:
I was literally staring at that before I made my comment and didn't notice... that was it... it's fixed.